### PR TITLE
Task 2 - Webpack setup

### PR DESCRIPTION
**Pull request by pavlobaranchuk**

### CODE REVIEW

**Review by IYeskov — APPROVED** [review]
Good job

File: src/js/function.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function Exams(data, marks) {
+  console.log(`Results:
+  Subject: ${data.js}, mark: ${marks[0]},
+  Subject: ${data.html}, mark: ${marks[1]},
+  Subject: ${data.css}, mark: ${marks[2]}.`);
+}
+
+export { Exams };
```
Comment (ihor4568) [inline src/js/function.js]: please use named export here. You can just export `Exams ` without curly braces

**Comment by ihor4568** [comment]
1 - Please add all your team mates as reviewers to your PRs
2 - Extra task: please try to use https://github.com/jantimon/html-webpack-plugin instead of creating html manually.

File: src/js/function.js (hunk 1 above)
Comment (pavlobaranchuk) [reply src/js/function.js]: Done

**Pull request merged**